Software/DumptruckUltra-MCU/hw/i2c/i2cBusManager.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * i2cBusManager.cpp
+ *
+ *  Created on: Mar 27, 2023
+ *      Author: surao
+ */
+#include "i2cBusManager.hpp"
+
+I2CBusManager::I2CBusManager(struct i2cPin_t *i2cPins) {
```

Comment:
Consistency: maybe drop struct?  Unneeded, unused in other contexts.

Reason: TLDR; `struct` and `class` are practically the same thing in C++.